intg/src/main/java/org/apache/atlas/model/instance/AtlasRelationship.java

```diff
@@ -71,6 +72,8 @@ public class AtlasRelationship extends AtlasStruct implements Serializable {
     public static final String KEY_BLOCKED_PROPAGATED_CLASSIFICATIONS = "blockedPropagatedClassifications";
     public static final String KEY_PROPAGATED_CLASSIFICATIONS         = "propagatedClassifications";
 
+    public static final String CUSTOM_RELATIONSHIP_INFO               = "customRelationshipInfo";
```

Comment:
what is this being used for?

Reply:
It is used to send some custom data to consumers apart from Atlas known fields, we are using it to send vertexId to ES doc Id mappings and some other metadata